antora-playbook-dev.yml

```
@@ -0,0 +1,15 @@
+site:
+  title: TinyMCE Documentation (Dev)
+  url: https://tiny.cloud/docs
```

Would this be better off using the staging site as well?
```suggestion
  url: https://staging.tiny.cloud/docs
```